ivy/functional/backends/torch/experimental/manipulation.py

```diff
@@ -156,11 +156,9 @@ def vsplit(
 
 def dsplit(
     ary: torch.Tensor,
-    indices_or_sections: Union[int, Tuple[int]],
+    indices_or_sections: Union[int, Tuple[int, ...]],
     /,
-    *,
-    out: Optional[torch.Tensor] = None,
-) -> torch.Tensor:
+) -> Tuple[torch.Tensor, ...]:
     return torch.dsplit(ary, indices_or_sections)
```

Comment:
Output should be converted to list to align with the rest of the backends

Reply:
Thanks, updated it